flow/flow.proto

```diff
@@ -149,7 +149,7 @@ message Flow {
 
 /* extra layers */
   layers.DHCPv4 DHCPv4 = 1000;
-  layers.DNS DNS = 1001;
+  repeated layers.DNS DNS = 1001;
```

Comment:
I won't use a an array here. I would prefer to just use one field, filling the questions and the answers as it is the same flow so questions/answers. Moreover changing the format will change the way we request the information for the API and from the data store.

Reply:
done